tests/config/test_config.py

```diff
@@ -164,6 +166,16 @@ def test_default_config(self):
         conf = config.ConfigManager(None, None)
         conf._validate_all()
 
+    def test_default_key_config(self):
+        """Test validating of the default key config."""
+        # We import qutebrowser.app so the cmdutils.register decorators run.
+        import qutebrowser.app
+        conf = keyconf.KeyConfigParser(None, None)
+        runner = runners.CommandRunner(win_id=0)
+        for sectname in configdata.KEY_DATA:
```

Comment:
Yeah, I thought about that as well - but since this is essentially one smoke test, I decided I didn't want it to count as ~200 tests :wink: